content/en/lotus/manage/chain-management.md

```diff
@@ -74,6 +74,20 @@ lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws
 
 {{< /alert >}}
 
+#### New Lightweight Snapshot Service
+
+We have soft launched a new Lightweight chain snapshot service which will be replacing the snapshots above in the future. More information about these snapshots can be found in [Notion](https://pl-strflt.notion.site/Lightweight-Filecoin-Chain-Snapshots-17e4c386f35c44548f5863afb7b5e024).
```

Comment:
add:
This snapshot is still considered experimental, and should not be used for production at this time. 
Please provide feedback in Filecoin Slack in `#fil-infra`